src/lib/components/Communities/CommunityHeader.js

```diff
@@ -0,0 +1,138 @@
+// This file is part of InvenioRDM
+// Copyright (C) 2020-2021 CERN.
+// Copyright (C) 2020-2021 Northwestern University.
+// Copyright (C) 2021 Graz University of Technology.
+//
+// Invenio RDM Records is free software; you can redistribute it and/or modify it
+// under the terms of the MIT License; see LICENSE file for more details.
+
+import React from 'react';
+import { Container, Image, Icon, Placeholder } from 'semantic-ui-react';
+import PropTypes from 'prop-types';
+
+class CommunityHeaderIcon extends React.Component {
+  render() {
+    return (
+      <Icon
+        className="ml-5"
+        link
+        color="grey"
+        onClick={() => console.log('TODO: Open community modal')}
+        name="setting"
+      />
+    );
+  }
+}
+
+class CommunityHeaderLoading extends React.Component {
+  render() {
+    return (
+      <Container className="community-header">
+        <Placeholder>
+          <Placeholder.Header image>
+            <Placeholder.Line />
+            <Placeholder.Line />
+          </Placeholder.Header>
+        </Placeholder>
+      </Container>
+    );
+  }
+}
+
+class CommunityHeaderInfo extends React.Component {
+  render() {
+    const { community, imagePlaceholderLink } = this.props;
+    const logoLink = community ? community?.links?.logo : imagePlaceholderLink;
+    return (
+      <Container className="community-header">
+        <Image size="mini" src={logoLink} /> {community?.metadata?.title}
+        <CommunityHeaderIcon />
+      </Container>
+    );
+  }
+}
+
+CommunityHeaderInfo.propTypes = {
+  imagePlaceholderLink: PropTypes.string.isRequired,
+  community: PropTypes.object,
+};
+
+CommunityHeaderInfo.defaultProps = {
+  community: undefined,
+};
+
+class EmptyCommunity extends React.Component {
+  render() {
+    const { error } = this.props;
+    return (
+      <Container className="community-header empty">
+        <span className={error && 'error-message'}>
```

Comment:
why not to use the `<Message>` component instead?

Reply:
I personally like it less than just red text:
With message:
![image](https://user-images.githubusercontent.com/15194802/154062494-5e48b731-7956-4cd2-9774-50bf4211aff9.png)

Just red text:
![image](https://user-images.githubusercontent.com/15194802/154025618-f580108c-31ac-44e2-b7b7-ba8037acb22c.png)

Reply:
Resolved no error handling needed here (at the moment)